tools/travis-before-install.sh

```diff
@@ -49,6 +49,7 @@ pip install --upgrade pip 'setuptools<49.2.0' wheel
 # requirement using `grep cython test_requirements.txt` instead of simply
 # writing 'pip install setuptools wheel cython'.
 pip install `grep cython test_requirements.txt`
+pip install `grep hpy.devel test_requirements.txt`
```

Comment:
is this needed for a good reason, or simply because we haven't released `hpy.devel` on PyPI? In theory the `setup_requires=['hpy.devel']` should be enough for setuptools to do the "right thing", isn't it?

Reply:
Well, I think that the comment about Cython applies to HPy as well: we probably want to ensure that we have a specific version of HPy and setup_requires may not be enough.

Reply:
> Well, I think that the comment about Cython applies to HPy as well

not necessarily: Cython has a weird way of integrating with setuptools: you need to call `cythonize` before the call to `setup()`, so setuptools doesn't even have a chance to install it even if you put in `setup_requires`.
HPy have a different approach, which is the same as cffi: it uses setuptool's entry points, so you don't need to be able to `import hpy.devel` before calling `setup()`.

> we probably want to ensure that we have a specific version of HPy and setup_requires may not be enough.

I think that `setup_requires=['hpy.devel==0.123']` would work, once we have proper releases